builders/xcm/remote-evm-calls.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Remote EVM Calls Through XCM
+description: How to do remote calls to Moonbeam EVM from other chains through XCM and tap into the Moonbeam EVM ecosystem from any parachain in the Polkadot ecosystem
+---
+
+# Remote EVM Calls Through XCM
+
+![Remote EVM Calls Banner](/images/builders/xcm/remote-evm-calls/xcmevm-banner.png)
+
+## Introduction {: #introduction}
+
+The [XCM-transactor pallet](/builders/xcm/xcm-transactor/){target=_blank} provides a simple interface to perform remote cross-chain calls through XCM. However, this does not consider the possibility of doing remote calls to Moonbeam's EVM, only to Substrate specific pallets (functionalities).
+
+Moonbeam's EVM is only accessible through the [Ethereum pallet](https://github.com/paritytech/frontier/tree/master/frame/ethereum){target=_blank}. Among many other things, this pallet handles validating transactions before going into the transaction pool. Then, it performs another validation step before inserting a transaction from the pool in a block. Lastly, it provides the interface through a `transact` function to execute a validated transaction. All these steps follow the same behavior as an Ethereum transaction in terms of structure and signature scheme.
+
+However, calling the [Ethereum pallet](https://github.com/paritytech/frontier/tree/master/frame/ethereum){target=_blank} directly through an XCM [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} is not feasible. Mainly because the dispatcher account for the remote EVM call (referred to as `msg.sender` in  Ethereum) does not sign the XCM transaction on the Moonbeam side. The XCM extrinsic is signed in the origin chain, and the XCM executor dispatches the call, through the [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} instruction, from a known caller linked to the sender in the origin chain. In this context, the Ethereum pallet will not be able to verify the signature and, ultimately, validate the transaction.
+
+To this end, the [Ethereum-XCM pallet](https://github.com/PureStake/moonbeam/tree/master/pallets/ethereum-xcm){target=_blank} was introduced. It acts as a middleware between the XCM [Transact](https://github.com/paritytech/xcm-format#transact){target=_blank} instruction and the [Ethereum pallet](https://github.com/paritytech/frontier/tree/master/frame/ethereum){target=_blank}, as special considerations need to be made when performing EVM calls remotely through XCM. The pallet performs the necessary checks and validates the transaction. Next, the pallet calls the Ethereum pallet to dispatch the transaction to the EVM. Due to how the EVM is accessed, there are some differences between regular and remote EVM calls.
```

Comment:
It might be nice to go through the entire flow of a remote EVM call in this paragraph, cause it has the latter half but just not the beginning like where the call is initiated from

Reply:
Ok I think it might be a good idea to include one of our world renown Happy Path diagrams 😅  - But yeah I agree

Reply:
I second this. I was going to suggest the same thing when reading this paragraph. Visuals always help =)

Reply:
I've added a diagram